modules/telemetry-showing-data-collected-from-the-cluster.adoc

```diff
@@ -6,28 +6,154 @@
+--data-urlencode 'match[]={__name__="nto_custom_profiles:count"}' \
+--data-urlencode 'match[]={__name__="openshift_csi_share_configmap"}' \
+--data-urlencode 'match[]={__name__="openshift_csi_share_secret"}' \
+--data-urlencode 'match[]={__name__="openshift_csi_share_mount_failures_total"}' \
+--data-urlencode 'match[]={__name__="openshift_csi_share_mount_requests_total"}' \
+--data-urlencode 'match[]={__name__="cluster:velero_backup_total:max"}' \
+--data-urlencode 'match[]={__name__="cluster:velero_restore_total:max"}' \
+--data-urlencode 'match[]={__name__="eo_es_storage_info"}' \
+--data-urlencode 'match[]={__name__="eo_es_redundancy_policy_info"}' \
+--data-urlencode 'match[]={__name__="eo_es_defined_delete_namespaces_total"}' \
+--data-urlencode 'match[]={__name__="eo_es_misconfigured_memory_resources_info"}' \
+--data-urlencode 'match[]={__name__="cluster:eo_es_data_nodes_total:max"}' \
+--data-urlencode 'match[]={__name__="cluster:eo_es_documents_created_total:sum"}' \
+--data-urlencode 'match[]={__name__="cluster:eo_es_documents_deleted_total:sum"}' \
+--data-urlencode 'match[]={__name__="pod:eo_es_shards_total:max"}' \
+--data-urlencode 'match[]={__name__="eo_es_cluster_management_state_info"}' \
+--data-urlencode 'match[]={__name__="imageregistry:imagestreamtags_count:sum"}' \
+--data-urlencode 'match[]={__name__="imageregistry:operations_count:sum"}' \
+--data-urlencode 'match[]={__name__="log_logging_info"}' \
+--data-urlencode 'match[]={__name__="log_collector_error_count_total"}' \
+--data-urlencode 'match[]={__name__="log_forwarder_pipeline_info"}' \
+--data-urlencode 'match[]={__name__="log_forwarder_input_info"}' \
+--data-urlencode 'match[]={__name__="log_forwarder_output_info"}' \
+--data-urlencode 'match[]={__name__="cluster:log_collected_bytes_total:sum"}' \
+--data-urlencode 'match[]={__name__="cluster:log_logged_bytes_total:sum"}' \
+--data-urlencode 'match[]={__name__="cluster:kata_monitor_running_shim_count:sum"}' \
+--data-urlencode 'match[]={__name__="platform:hypershift_hostedclusters:max"}' \
+--data-urlencode 'match[]={__name__="platform:hypershift_nodepools:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_unhealthy_bucket_claims:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_buckets_claims:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_unhealthy_namespace_resources:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_namespace_resources:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_unhealthy_namespace_buckets:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_namespace_buckets:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_accounts:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_usage:max"}' \
+--data-urlencode 'match[]={__name__="namespace:noobaa_system_health_status:max"}' \
+--data-urlencode 'match[]={__name__="ocs_advanced_feature_usage"}'
```

Comment:
missed `os_image_url_override:sum` according to:

https://github.com/openshift/cluster-monitoring-operator/blob/release-4.12/manifests/0000_50_cluster-monitoring-operator_04-config.yaml

since the new metrics would be added after the doc is merged, not sure if we have a better solution for the doc, otherwise we have to update the doc if new metrics  added later